### PR TITLE
Fix: grafana - var name and firewall check

### PR DESCRIPTION
**Pull request by thiagocardozo**
The process running check for firewall configuration is not common in other roles.
Besides that, Ansible is also complaining that this type of substitution is not supposed to be done in conditionals, therefore
ending in error as {{ item }} is always nil.

### CODE REVIEW

**Review by strus38 — APPROVED** [review]

**Review by neilmunday — APPROVED** [review]

File: roles/prometheus/vars/main.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ---
-prometheus_role_version: 1.2.2
\ No newline at end of file
+prometheus_role_version: 1.2.3
```
Comment (oxedions) [inline roles/prometheus/vars/main.yml:2]: Why did you updated prometheus role version while the role updated was Grafana ?

**Review by oxedions — CHANGES_REQUESTED** [review]
Thank you for this PR.
A small comment regarding role version upgrade.
Also, you forgot to update main changelog 🙂

File: roles/prometheus/vars/main.yml (hunk 1 above)
Comment (thiagocardozo) [reply roles/prometheus/vars/main.yml:2]: I was working on both roles simultaneously and made a confusion.

**Review by neilmunday — APPROVED** [review]

**Review by strus38 — CHANGES_REQUESTED** [review]
please fix since it seems that github actions are not passing.

**Review by neilmunday — APPROVED** [review]

**Pull request closed without merging**